Helper/AnalyticsHelper.php

```diff
@@ -0,0 +1,380 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\Helper;
+
+use Algolia\AlgoliaSearch\Helper\Entity\CategoryHelper;
+use Algolia\AlgoliaSearch\Helper\Entity\PageHelper;
+use Algolia\AlgoliaSearch\Helper\Entity\ProductHelper;
+use AlgoliaSearch\Analytics;
+
+class AnalyticsHelper extends Analytics
```

Comment:
How about use `algoliaHelper` to "create" and get instance of `AlgoliaSearch\Analytics`?
If I didn't overlook something, this class uses only `request()` method of the `AlgoliaSearch\Analytics` and I feel it's not needed to extend the class completely.

By switching the responsibility of creation of `AlgoliaSearch\Analytics` to `AlgoliaHelper` we keep the right responsibilities in right classes. What do you think of that?
At the same time we'd reduce coupling with Algolia API client in general.

Reply:
Admittedly, I just tried to create an instance of AlgoliaSearch\Analytics but then was remembered why I extended the API in the first place. The request() method is protected and not accessible outside of the class.

Reply:
/o\ Makes sense then, I had no idea the Analytics class has this limitation. All good then 👍 :)